src/main/java/uk/gov/companieshouse/items/orders/api/ItemsApiApplication.java

```diff
@@ -7,6 +7,15 @@
 public class ItemsApiApplication {
 
 	public static final String APPLICATION_NAMESPACE = "items.orders.api.ch.gov.uk";
+	
+    public static final String COMPANY_NUMBER_LOG_KEY = "company_number";
+    public static final String REQUEST_ID_LOG_KEY = "request_id";
+    public static final String CERTIFICATE_ID_LOG_KEY = "certificate_id";
+    public static final String USER_ID_LOG_KEY = "user_id";
+    public static final String STATUS_LOG_KEY = "status";
+    public static final String ERRORS_LOG_KEY = "errors";
+    public static final String IDENTITY_LOG_KEY = "identity";
+    public static final String REQUEST_ID_HEADER_NAME = "X-Request-ID";
 
```

Comment:
Is it a convention to place these logging keys (and header name) in the application class? Maybe they could be in a constants class of their own?

Reply:
Happy to place in their own class, I was following on from the namespace declaration as that is also used. I'll extract all of them to a LoggingConstants class